mqr/baseline_ussd.py

```diff
@@ -0,0 +1,119 @@
+import logging
+
+import aiohttp
+
+import vaccine.healthcheck_config as config
+from mqr.utils import rapidpro
+from vaccine.base_application import BaseApplication
+from vaccine.states import Choice, ChoiceState, EndState
+from vaccine.utils import (
+    HTTP_EXCEPTIONS,
+    normalise_phonenumber,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def get_rapidpro():
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Authorization": f"Token {config.RAPIDPRO_TOKEN}",
+            "Content-Type": "application/json",
+            "User-Agent": "baseline-ussd",
+        },
+    )
+
+
+class Application(BaseApplication):
+    START_SURVEY = "survey_start"
+
+    async def survey_start(self):
+        if config.RAPIDPRO_URL and config.RAPIDPRO_TOKEN:
+            msisdn = normalise_phonenumber(self.inbound.from_addr)
+            if msisdn.startswith("+"):
+                urn = ' f"whatsapp:{msisdn.lstrip(' + ')}"'
```

Comment:
I think we have a few extra single quotes and spaces here: 
urn = f"whatsapp:{msisdn.lstrip('+')}"

Reply:
The if statement is also not necessary, the lstrip will just not do anything if + isn't there